specification/overview.md

```diff
@@ -26,21 +26,58 @@ This specification is designed to be used by three major groups:
+* Columns should have a single purpose, with clear and concise names and values.
+* Data should be human-readable, free from jargon, using simple understandable terms, and be approachable.
+* Naming and terms used should be carefully considered to avoid using terms for which the definition could be confused by the reader. If a term must be used which has either an unclear or multiple definitions, it should be clarified in the [glossary](#glossary).
+* The specification should provide all of the data elements necessary for the FinOps capabilities.
+
+### Provider-Neutral Approach by Default
+
+* While the schema, naming, terminology, and attributes of many providers are reviewed during development, this specification aims to be provider-neutral.
+* Contributors must take care to ensure the specification examines how each decision relates to each of the major cloud providers and SaaS vendors, not favoring any one.
+* In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
 
 ### Extensibility
 
-The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs). The specification however aims to be extensible to SaaS products and other types of cost datasets. Future versions of the specification will look to expand the content to support a broader set of prioritized FinOps capabilities.
+* The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs).
+* The specification, however, aims to be extensible to SaaS products and other types of cost datasets.
+* Future versions of the specification will look to expand the content to support a broader set of prioritized FinOps capabilities.
+
+## Design Notes
+
+### Optimize for data analysis
+
+* Columns should be optimized for data analysis (aggregations and breakdowns) at scale and should not require splitting or parsing.
+* Data that cannot be represented in predefined columns (e.g., tags) should use a simplified JSON object that does not use nested objects and arrays, and is parseable within spreadsheet applications (e.g. Excel, Google Sheets.)
+* Avoid complex JSON structures when an alternative columnar structure is possible.
+* Avoid creating a technical barrier for parsing and processing any FOCUS data value.
+* FOCUS should include all data necessary as the system of record for cost and usage data and scenarios that require this dataset.
+* Minimize the need for expensive joins at scale for non-technical personas and avoid duplicating data for scenarios that require engineering support where complex joins are more feasible (e.g., forecasting).
```

Comment:
The two sections of this sentence feel contradictory to me.  If that is the very point of the sentence, then perhaps it could be more explicit.  Like:
```suggestion
* Minimize the need for expensive joins at scale for non-technical personas while also avoiding the duplication of data for scenarios that require engineering support where complex joins are more feasible (e.g., forecasting).
```